global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.100",
+    "version": "7.0.200",
```

Comment:
Could this target `7.0.202`?

Reply:
Definitely possible on `dev`; it's less important to target the latest TFM precisely for app packages, since .NET platform assemblies are bundled by the host (`seqcli`) rather than with the app itself.